src/main/java/com/teamwizardry/librarianlib/features/utilities/JsonGenerationUtils.kt

```diff
@@ -31,7 +31,7 @@ fun getPathPrefix(modid: String) = paths[modid] ?: "src/main/resources/assets"
 
 private val paths = mutableMapOf<String, String>()
 
-private operator fun String.unaryPlus() = this.replace("/", File.separator)
+operator fun String.unaryPlus() = this.replace('/', File.separatorChar)
```

Comment:
NO. PRIVATE. KEEP IT THAT WAY.

Reply:
Could you please elaborate on why? :)

Reply:
because we don't want to define a global unary plus for strings.

Reply:
It's not exactly global, it's `com.teamwizardry.librarianlib.features.utilities.unaryPlus`, but I guess I'll keep it private